video/rest/compositions/get-composition-media-file/get-composition-media-file.5.x.rb

```diff
@@ -14,10 +13,12 @@
 
 uri = "https://video.twilio.com/v1/Compositions/#{composition_sid}/Media?Ttl=3600"
 
-response = client.request("video.twilio.com", 433, 'GET', uri)
+response = client.request('video.twilio.com', 433, 'GET', uri)
 
 media_location = response.body['redirect_to']
 # For example, write the composition media to a local file
-open('myFile.mp4', "wb") do |file|
-  file << open(media_location).read
+response = Net::HTTP.get_response(URI.parse(media_location))
+
+open('myFile.mp4', 'wb') do |file|
+  file.write(response.body)
```

Comment:
Changed the file download implementation to use a safe strategy instead of `open()` from `open-uri`